<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>151</width>
      <height>571</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>开始游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>载入游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>保存游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseButton">
        <property name="text">
         <string>暂停游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resumeButton">
        <property name="text">
         <string>继续游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restartButton">
        <property name="text">
         <string>重新开始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>退出游戏</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="loadAction"/>
    <addaction name="saveAction"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="gameMenu">
    <property name="title">
     <string>游戏</string>
    </property>
    <addaction name="startAction"/>
    <addaction name="pauseAction"/>
    <addaction name="resumeAction"/>
    <addaction name="restartAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="gameMenu"/>
  </widget>
  <action name="loadAction">
   <property name="text">
    <string>载入游戏</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>保存游戏</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>退出游戏</string>
   </property>
  </action>
  <action name="startAction">
   <property name="text">
    <string>开始游戏</string>
   </property>
  </action>
  <action name="pauseAction">
   <property name="text">
    <string>暂停游戏</string>
   </property>
  </action>
  <action name="resumeAction">
   <property name="text">
    <string>继续游戏</string>
   </property>
  </action>
  <action name="restartAction">
   <property name="text">
    <string>重新开始</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
